app/src/main/java/com/example/internetshop/presentation/activity/fragments/CategoriesFragment.kt

```diff
@@ -22,6 +23,18 @@ class CategoriesFragment : BaseFragment() {
         component.inject(this)
     }
 
+    override fun getTitle(): String {
```

Comment:
all three methods should be in body style

Reply:
all fixed